.github/CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 /docs/ @apollographql/docs
 /.changesets/ @apollographql/docs
-/apollo-federation/ @dariuszkuc @sachindshinde @goto-bus-stop @SimonSapin @lrlna @TylerBloom @duckki
+/apollo-federation/ @apollographql/fed-core
```

Comment:
@goto-bus-stop @SimonSapin @lrlna do you want to continue to be pinged on the federation PRs?